pkg/scrape/darkroomvr.go

```diff
@@ -0,0 +1,130 @@
+package scrape
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/gocolly/colly"
+	"github.com/nleeper/goment"
+	"github.com/thoas/go-funk"
+	"github.com/xbapps/xbvr/pkg/models"
+)
+
+func DarkRoomVR(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out chan<- models.ScrapedScene) error {
+	defer wg.Done()
+	scraperID := "darkroomvr"
+	siteID := "DarkRoomVR"
+	logScrapeStart(scraperID, siteID)
+
+	sceneCollector := createCollector("darkroomvr.com")
+	siteCollector := createCollector("darkroomvr.com")
+
+	sceneCollector.OnHTML(`html`, func(e *colly.HTMLElement) {
+		sc := models.ScrapedScene{}
+		sc.SceneType = "VR"
+		sc.Studio = "DarkRoomVR"
```

Comment:
This is a Virtual Taboo site.

```suggestion
		sc.Studio = "VirtualTaboo"
```